pkgs/applications/editors/your-editor/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "your-editor";
-  version = "1203";
+  version = "1205";
```

Comment:
This should be using the unstable-XXX-XX-XX version format.

Reply:
it is versioned despite not being tagged: https://github.com/kammerdienerb/yed/blob/master/src/version.h
the latest is actually 1206 now

It's probably a good idea to open an issue upstream to tag the releases so we don't have to keep track of the rev

Reply:
Even if yed is versioned?
https://github.com/kammerdienerb/yed/blob/master/src/version.h#L4 
also it just got bumped to 1206, so is it ok to close this pr and reopen it in some hours after i test 1206?

Reply:
Ok I will bring it up, thank you